client/src/Dataset/Details/Details.tsx

```diff
@@ -56,53 +62,86 @@ const Details: React.FC<{ id: number }> = ({ id }) => {
         );
     }
     return (
-        <Grid container>
-            <Grid item xs={3} sm={2}>
-                <Paper className={classes.panel}>
-                    <TitleBar
-                        title={dataset.name}
-                        imagesCount={imagesCount}
-                        pagesCount={pagesCount}
-                    />
-                    <ButtonsBar
-                        generateAction={generateAction}
-                        scanAction={scanAction}
-                        importAction={importAction}
-                        exportAction={exportAction}
-                        resetMetadataAction={resetMetadataAction}
-                    />
-
-                    <SubdirForm
-                        folder={folder}
-                        setFolder={setFolder}
-                        subdirectories={subdirectories}
-                    />
-
-                    <Paper>
-                        <Box p={1}>
-                            <FilterForm filters={filters} />
-                        </Box>
-                    </Paper>
-                </Paper>
+        <Grid container className={classes.root}>
+            <Grid item lg={2} md={3} sm={5} xs={6}>
+                <div ref={sidebarRef}>
+                    <Drawer
+                        className={classes.drawer}
+                        variant="permanent"
+                        classes={{
+                            paper: classes.drawerPaper,
+                        }}
+                        anchor="left"
+                    >
+                        <div
+                            id="dragger"
+                            onMouseDown={(event: React.MouseEvent) => {
```

Comment:
if you pass the default argument to the handler function then there's no need to create new function
```suggestion
                            onMouseDown={handleMouseDown}
```